_posts/2023-11-21-trino-summit-2023-nears-lineup.md

```diff
@@ -0,0 +1,174 @@
+Delta Lake, Trino, Apache Kafka, and Open Policy Agent (OPA). This talk will
+discuss the challenges involved, provide solutions and considerations for each,
+and end with a demo of Raft’s Data Fabric. The talk will focus on
+a plugin for Trino, developed by Raft, that uses OPA as a policy engine to
+provide fine-grained access control at query time based on a user’s JWT passed
+along with the query.
+
+## Treasure Data: Secure exchange SQL
+
+Secure Exchange SQL is a production Data Clean Room service deployed at Treasure
+Data, which leverages Trino and differential privacy technology to enable
+cross-company data analysis while mitigating the risk of privacy breaches.
+In their session, they'll introduce the concept of differential privacy and
+discuss the privacy protection methods that need to be implemented during SQL
+processing. To minimize changes to Trino's codebase, they employed approaches of
+SQL rewriting and validation at the logical plan level. They'll explain these
+methods and provide some practical use cases of their Data Clean Room.
+
+## Zomato: Powering data marts through the Trino Iceberg connector
+
+It's a common theme in the Trino community - Zomato recently migrated from a
+traditional data warehouse to a Trino-powered data lakehouse in conjunction with
+Iceberg. They'll be discussing how this has enabled their analytics to run
+better than ever, including periodic updates to their data marts and tackling
+the challenges involved in maintaining Iceberg tables.
+
+## Bazaar: Powering Bazaar`s business operations using Trino
+
+Bazaar's talk will discuss how they leverage Trino's capabilities to optimize
+data analysis and support data-driven decision-making. The talk specifically
+explores including real-time data querying across multiple sources and
+performance optimization, illustrating Trino's role in Bazaar's data-centric
+strategies. This presentation provides in-depth insights for individuals
+well-versed in Trino, shedding light on the platform's transformative impact on
+enhancing e-commerce operations.
+
+## Preset: Visualizing Trino with Superset
+
+Preset will be diving into the "last mile" of the modern data stack and
+show you how to query and visualize data pulled from Trino with Apache Superset
+and/or Preset. Specifically, they'll discuss into things like Trino's federated
+query support (a common wish for Superset users) and how Superset can support
+near-real-time analytics for Trino users. They'll also give a demo of connecting
+to Trino, building SQL Queries, designing charts and dashboards, and other ways
+to gain insight and stay on top of your data.
+
+## VAST: The VAST database catalog
+
+The VAST Database connector for Trino was open-sourced this year! They'll be
+discussing the architecture of VAST and the connector, the purpose and major use
+cases for it, and demonstrate the workflows surrounding the VAST Database in the
+Trino ecosystem.
```

Comment:
Blog post needs a conclusion ... something like .. more sessions will be announced and there are already great so dont forget to register and see you there.